jsonrpc/src/main/scala/scala/meta/jsonrpc/SourceLocation.scala

```diff
@@ -0,0 +1,17 @@
+package scala.meta.jsonrpc
+
+class SourceLocation(val file: String, val line: Int) {
```

Comment:
Source location for error messages is too helpful for debugging that I couldn't resist to keep it out.